tests/shell/shell_recorder/tutorial_wrapper/markdown_shell_recorder.sh

```diff
@@ -29,6 +29,15 @@ writeBlock()
 	fi
 	COMMAND=$(sed s/sudo\ //g <<< "$COMMAND")
 	while read -r cmd; do
+		match_command='(kdb[ \t]+(set(meta)?|rm)|kdbSet|keySetName)'
```

Comment:
Other variables in this file are written in uppercase. We should introduce shell coding guidelines in doc/CODING.md though.

Reply:
Yeah, basically we use all kinds of variable styles all over the place inside shell scripts. Since this is already the case, I used one of the most common styles for scripting languages.

Reply:
Yes, we definitely need some style guide. Breaking the style within a file can be very confusing. (People think these variables are used differently than the other ones.)